### PR TITLE
add materials about swift

### PR DESCRIPTION
**Pull request by ELezov**
## Hacktoberfest notes:

- due to volume of submissions, we may not be able to review PRs that do not pass tests and do not have informative titles.
- please read our [contributing guidelines](/CONTRIBUTING.md)
- be sure to check the output of Travis-CI for linter errors
- if this is your first open source contribution, make sure it's not your last!

## What does this PR do?
Add Resource(s) | Remove Resource(s) | Add info | Improve Repo

### Checklist:
- [x] Not a duplicate
- [x] Included author(s) if appropriate
- [x] Lists are in alphabetical order
- [x] Needed indications added (PDF, access notes, under construction)

### CODE REVIEW

File: free-programming-books-ru.md [hunk 1]
```diff
@@ -235,6 +236,7 @@
 * [Руководство по Java Core](http://proselyte.net/tutorials/java-core) - Евгений Сулейманов
 * [Руководство по Servlets](http://proselyte.net/tutorials/servlets) - Евгений Сулейманов
 * [Руководство по языку программирования Java](https://metanit.com/java/tutorial) - Евгений Попов
+* [Самоучитель по Java в игровой форме](https://javarush.ru)
```
Comment (eshellman) [inline free-programming-books-ru.md:239]: not a book. Maybe the Russsian page has been lax about separating genres. What do you think?

File: problem-sets-competitive-programming.md [hunk 2]
```diff
@@ -23,7 +24,6 @@
 * [Coderbyte](https://coderbyte.com)
 * [Codewars](http://www.codewars.com)
 * [Codingame](https://www.codingame.com/start)
-* [COCI](https://hsin.hr/coci/)
```
Comment (eshellman) [inline problem-sets-competitive-programming.md:26]: you're right!

File: free-programming-books-ru.md (hunk 1 above)
Comment (ELezov) [reply free-programming-books-ru.md:239]: Yes, you are right, it's not a book. It's my fail, i will fix it.

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**